classes/patterns/class-sideimagewithtextandcta.php

```diff
@@ -31,13 +31,16 @@ public static function get_config( $params = [] ): array {
 		$media_link           = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
 		$media_position_class = array_key_exists( 'media_position', $params ) && 'right' === $params['media_position'] ? 'has-media-on-the-right' : '';
 		$title_placeholder    = $params['title_placeholder'] ?? '';
+		$background_color     = $params['background_color'] ?? null;
+		$background_attribute = $background_color ? ',"backgroundColor":"' . $background_color . '"' : '';
+		$background_classes   = $background_color ? 'has-' . $background_color . '-background-color has-background' : '';
 
 		return [
 			'title'      => __( 'Side image with text and CTA', 'planet4-blocks-backend' ),
 			'categories' => [ 'planet4' ],
 			'content'    => '
-				<!-- wp:media-text {"mediaLink":"' . $media_link . '","mediaType":"image","className":"block ' . $media_position_class . '"} -->
-					<div class="wp-block-media-text alignwide is-stacked-on-mobile block ' . $media_position_class . '">
+				<!-- wp:media-text {"mediaLink":"' . $media_link . '","mediaType":"image","className":"block ' . $media_position_class . '"' . $background_attribute . '} -->
+					<div class="wp-block-media-text alignwide is-stacked-on-mobile block ' . $media_position_class . $background_classes . '">
```

Comment:
Missing a space between `$media_position_class` and `$background_classes` when both are defined